nerfstudio/model_components/ray_samplers.py

```diff
@@ -501,7 +501,7 @@ class ProposalNetworkSampler(Sampler):
 
     def __init__(
         self,
-        num_proposal_samples_per_ray: Tuple[int] = (64,),
+        num_proposal_samples_per_ray: Tuple[int, ...] = (64,),
```

Comment:
why this?

Reply:
Typing issue since if you look at the nerfacto config, the num_proposal_samples_per_ray is a tuple (256, 256)